src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -2836,21 +2836,18 @@ def create_new_cluster_with_monitoring_aad_auth(self, resource_group, resource_g
 
         if data_collection_settings:
              self.cmd(get_cmd, checks=[
-                self.check('properties.dataSources.extensions[0].name', f'ContainerInsightsExtension')
-            ])
-             self.cmd(get_cmd, checks=[
-                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.interval', f'1m')
-            ])
-             self.cmd(get_cmd, checks=[
-                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.namespaceFilteringMode', f'Include')
-            ])
-             self.cmd(get_cmd, checks=[
-                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.namespaces[0]', f'kube-system')
+                self.check('properties.dataSources.extensions[0].name', f'ContainerInsightsExtension'),
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=75964432&view=results).

Reply:
this is because https://github.com/Azure/azure-cli/pull/26793/files has not been merged yet, will retrigger the test after that merged.

Reply:
re ran the tests, now all looks good